### DISCUSSION TITLE
Why can not support BTC or other chains?

### DISCUSSION THREAD

**Discussion by BigVoctor** [Web3Auth/Web3Auth#432] [Feature Requests]
Hi, there! I have some questions: 
1. Why not support BTC or other chains?
2. Can you back up the root private key? If you can back up the root private key, you should be able to support more chains.
3. If we use your SDK, can we decide the content of the string to be backed up, such as backing up our encrypted private key. If so, is there a limit to the length of the string?

**Comment by itsinz** [comment]
@yashovardhan
